app/views/assignments/index.haml

```diff
@@ -1,32 +1,33 @@
 %h1.title #{@roster.name} On-Call
+= render partial: 'current'
+= render partial: 'your_assignments' if @assignments.present?
+
 %h2.title= @month_date.strftime '%-B %G'
+%table#calender
+  %thead
+    %tr
+      - WEEKDAYS.each do |day|
+        %th.cal-head= day
 
-%table#calendar
-  %tr
-    - WEEKDAYS.each do |day|
-      %th= day
   - @weeks.each do |week|
-    %tr
+    %tr.cal-row
       - week.each do |day|
-        %td
-          - assignment = Assignment.in(@roster).on(day)
-          %div{ class: ('lightgrey' if day.month != @month_date.month) }
-            = day.strftime '%-m/%-d'
-          .assignment
-            - if assignment.present?
-              = link_to assignment.user.last_name,
-                edit_roster_assignment_url(@roster, assignment)
-              .arrows
-                - if day == assignment.start_date
-                  %span.left &#8594;
-                - if day == assignment.end_date
-                  %span.right &#8592;
-            - else
-              .slightly-red= link_to 'None assigned',
-                new_roster_assignment_url(@roster, date: day)
-  %span.left= link_to @month_date.last_month.strftime('%B %Y'),
+        - assignment = Assignment.in(@roster).on(day)
+        %td.cal-cell{ class: cal_day(day: day) }
+          .date
+            = day.strftime '%-d'
+            = if day.strftime('%-d') == '1'
```

Comment:
I think here `if day.day == 1` might be clearer, rather than formatting a date as a string in order to compare it to another string.

Perhaps most clear would be `if day == day.month.beginning_of_month`, but that's probably overkill.